app/models/manifest.rb

```diff
@@ -1,7 +1,9 @@
 require 'date'
 
 class Manifest < ActiveRecord::Base
-  validates :tracking_number, presence: true
+  include Searchable
+
+  validate :tracking_number, :validate_tracking_number_unique
```

Comment:
I tried `uniqueness: true` at first too, but since `tracking_number` isn't a proper database column, AR doesn't know how to generate SQL for it.